uber/site_sections/accounts.py

```diff
@@ -19,7 +19,7 @@ def index(self, session, message=''):
             'all_attendees': sorted([
                 (id, '{} - {}{}'.format(name.title(), BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
                 for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
-                                                              .filter(Attendee.email != '').all()
+                                                              .filter(Attendee.email != '').filter(Attendee.status != INVALID_STATUS).all()
```

Comment:
Whoops, sorry. In that case I tend to use the or_ syntax, which you can find an example of in the first answer here: https://stackoverflow.com/questions/7942547/using-or-in-sqlalchemy